### PR TITLE
AAuser, when I open SM for the first time, I see a /onboarding page empty

### PR DESCRIPTION
**Pull request by hypervillain**
Read onboarding state from localstorage and redirect to /onboarding.
Once onboarding page is loaded, add a line in localstorage that states user did see the onboarding

### CODE REVIEW

File: packages/slice-machine/pages/onboarding.tsx [hunk 1]
```diff
@@ -1,65 +1,15 @@
...
-      </Fragment>
-    )}
-  </Box>
-);
+import { LocalStorageKeys } from "@lib/consts";
 
 export default function Changelog() {
-  const { data } = useSwr("/api/auth/validate", fetcher);
+  useEffect(() => {
+    localStorage.setItem(LocalStorageKeys.isOnboarded, "yes");
```
Comment (Miniplop) [inline packages/slice-machine/pages/onboarding.tsx]: can we use true boolean here instead of "yes" ?

**Review by Miniplop — COMMENTED** [review]
Nice, one little mistake and you can pass this ticket to need test. Can you change the PR title to the User story title ? :)

File: packages/slice-machine/pages/onboarding.tsx (hunk 1 above)
Comment (hypervillain) [reply packages/slice-machine/pages/onboarding.tsx]: I don't think you can store booleans in local storage

Comment (hypervillain) [reply packages/slice-machine/pages/onboarding.tsx]: do you mean you want the string to have val "true"?

Comment (Miniplop) [reply packages/slice-machine/pages/onboarding.tsx]: Yes good for me :)

File: packages/slicemachine-core/src/filesystem/manifest.ts [hunk 2]
```diff
@@ -46,23 +46,6 @@ export function retrieveManifest(cwd: string): FileContent<Manifest> {
...
-  if (maybeSMFile.exists === false) return null;
-  if (!maybeSMFile.content?.apiEndpoint) return null;
-
-  const repoUrl = new URL(maybeSMFile.content.apiEndpoint);
-  const correctBase = repoUrl.hostname.includes(baseUrl.hostname);
-  if (correctBase === false) return null;
-
-  return repoUrl.hostname.split(".")[0];
-}
-
```
Comment (MarcMcIntosh) [inline packages/slicemachine-core/src/filesystem/manifest.ts]: why delete this?

**Review by MarcMcIntosh — CHANGES_REQUESTED** [review]

File: packages/slicemachine-core/src/filesystem/manifest.ts (hunk 2 above)
Comment (hypervillain) [reply packages/slicemachine-core/src/filesystem/manifest.ts]: I didn't like the name of the function

Comment (MarcMcIntosh) [reply packages/slicemachine-core/src/filesystem/manifest.ts]: do you have any ideas what a better name would be?

Comment (hypervillain) [reply packages/slicemachine-core/src/filesystem/manifest.ts]: Sorry it wasn't clear: I was joking, this was a manual merge issue

**Comment by Heyrows** [comment]
Should be from a branch 0.1.2

**Review by MarcMcIntosh — APPROVED** [review]

File: packages/slice-machine/tests/project/cypress/integration/0-is-running/00-onboarding.spec.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  });
+
+  it('should redirect to /onboarding when is-onboared is not in local storage', () => {
+    cy.visit('/')
+    cy.url().should('eq', 'http://localhost:9999/onboarding')
+    cy.wait(1000)
+    cy.getLocalStorage("is-onboarded").should('eq', 'true')
+  })
+
+  it('when is-onboarded is in loal storge it should not redirect', () => {
```
Comment (Miniplop) [inline packages/slice-machine/tests/project/cypress/integration/0-is-running/00-onboarding.spec.js]: typo storge and loal

**Review by Miniplop — APPROVED** [review]

File: packages/slice-machine/tests/project/cypress/integration/0-is-running/00-onboarding.spec.js (hunk 3 above)
Comment (MarcMcIntosh) [reply packages/slice-machine/tests/project/cypress/integration/0-is-running/00-onboarding.spec.js]: fixed :)

**Pull request merged**